### PR TITLE
Upload macOS binaries for e2e testing

### PR DESCRIPTION
**Pull request by adaszko**
This is to facilitate testing on dev’s machines.

### CODE REVIEW

File: .github/workflows/frontend-testing.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
     branches: [master]
 
 jobs:
-  release-x86_64-linux-binaries:
+  upload-x86_64-linux-binaries:
```
Comment (cloudhead) [inline .github/workflows/frontend-testing.yml:8]: Why are we calling these `upload-*`? "release" seemed more correct, since we're also building.

Comment (adaszko) [reply .github/workflows/frontend-testing.yml:8]: I did not want those to be confused with creating GitHub releases

Comment (cloudhead) [reply .github/workflows/frontend-testing.yml:8]: Sure, but it's creating a new build/release. I guess if you're worried about confusion with github release, you could call it `release-gcloud-*`

Comment (adaszko) [reply .github/workflows/frontend-testing.yml]: Renamed accordingly

**Pull request merged**